public/data/cambodia/about.md

```diff
@@ -0,0 +1,17 @@
+# About PRISM
+
+Platform for Real-Time Impact and Situation Monitoring (PRISM) is a web-based platform which supports the National Committee for Disaster Management (NCDM), other government agencies and partners to rapidly assess climate hazard risks and their impact, to inform disaster risk management and social protection programs. PRISM simplifies the integration of geospatial data on hazards, such as droughts and floods, along with information on socioeconomic vulnerability (IDPoor) to present decision makers with actionable information on vulnerable populations exposed to hazards.
+
+## NCDM contact info
+
+NCDM's public webiste is accessible at: <a href="http://ncdm.gov.kh/" target="_blank">http://ncdm.gov.kh/</a>
+The general NCDM email address is: <a href="mailto:administration@ncdm.gov.kh" target="_blank">administration@ncdm.gov.kh</a>  
+
+
+<img src="data/cambodia/ncdm_l3.png" width="200" height="200">
```

Comment:
@JorgeMartinezG can you reduce the size (in mb) of the image? It will help it load faster

Reply:
> @JorgeMartinezG can you reduce the size (in mb) of the image? It will help it load faster

done